### DISCUSSION TITLE
Solaris 10 x86 not recognizing keyboard at installation (M1) [Emulation]

### DISCUSSION THREAD

**Discussion by Walid5648** [utmapp/UTM#5449] [UTM Help (macOS)]
**Describe the issue**  
I am trying to install Solaris 10 on UTM using emulation to install the x86 version. the first screen that appears to select Solaris install the keyboard works to select it, but past that screen the keyboard doesn't work anymore and i have tried multiple external USB keyboards but still nothing, because of the timer i get stuck at the select the keyboard layout screen.

![SCR-20230705-iww](https://github.com/utmapp/UTM/assets/92256256/6013efda-4976-4755-96e8-333892552a9f)

And this is where the keyboard stops responding at all:

![SCR-20230706-13j](https://github.com/utmapp/UTM/assets/92256256/991a78cf-3f38-49f4-ad14-2cd6f1c3c1e8)

So we get stuck in here forever:

![SCR-20230706-10b](https://github.com/utmapp/UTM/assets/92256256/de39e867-1be5-48bf-aac8-b0c42ccd31a1)

**Configuration**  

* UTM Version: Version 4.2.5 (81)
* macOS Version: Ventura 13.4.1
* Mac Chip M1 pro

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/11962790/debug.log)

VM config:
[config.plist.zip](https://github.com/utmapp/UTM/files/11957885/config.plist.zip)

**Comment by tifasoftware** [comment]
It might be a USB controller issue, try downgrading to usb 1 or ps2

**Comment by Walid5648** [comment]
I have tried that but nothing. isn't my mac keyboard supposed to work
anyway?
edit: also there is not usb 1 option to emulate in UTM
